components/responsive-image/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+function ResponsiveImage( { src, alt, naturalWidth, naturalHeight } ) {
+	const imageStyle = {
+		paddingBottom: ( naturalHeight / naturalWidth * 100 ) + '%',
+	};
+	return (
+		<div className="components-responsive-image">
+			<div style={ imageStyle } />
+			<img src={ src } className="components-responsive-image__image" alt={ alt } />
```

Comment:
In the future, or perhaps even now, we may want to make this component a bit more generic as a responsive panel. The padding technique is rarely used with images, except in this case where we don't know the rendered width, and often more for background images or video elements. It could also help avoid the need to overwhelm the props with individual `img` attributes we want to assign, if we instead accepted the `img` as a child.

Not too concerned about this being addressed here.